openmc/tallies.py

```diff
@@ -1281,6 +1281,23 @@ def get_pandas_dataframe(self, filters=True, nuclides=True,
                     filter_bins = np.tile(filter_bins, tile_factor)
                     df[filter.type + ' [MeV]'] = filter_bins
 
+                # mu, polar, and azimuthal
+                elif filter.type in ['mu', 'polar', 'azimuthal']:
+                    bins = filter.bins
+                    num_bins = filter.num_bins
```

Comment:
Could probably just put `num_bins` in the `range()` expression directly since that's the only place it's used